### PR TITLE
Modifying homepage to use search component

### PR DESCRIPTION
**Pull request by andysellick**
* Replacing existing homepage search box with search component from static
* New component designed to exactly match original (for now) so users should be unaware of change
* Relies upon https://github.com/alphagov/static/pull/1079 - must not be merged until that PR is approved and deployed.

https://trello.com/c/QUGWKwTe/431-unify-search-boxes-as-component

screen shot 2017-07-10 at 09 05 54

screen shot 2017-07-10 at 09 06 14

**Older browsers**

IE7:

screen shot 2017-07-27 at 14 29 10

IE8:

screen shot 2017-07-27 at 14 25 53

IE9:

screen shot 2017-07-27 at 14 28 30

### CODE REVIEW

File: app/assets/stylesheets/views/_homepage.scss [hunk 1]
```diff
@@ -157,125 +157,7 @@ body.homepage {
...
-        .submit {
-          @include box-sizing(border-box);
-          height: $search-button-size;
-          width: $search-button-size;
 
-          cursor: pointer;
-
-          overflow: hidden;
-          text-indent: -5000px;
-          background: $black image-url("homepage/search-button.png") no-repeat 2px 50%;
```
Comment (fofr) [inline app/assets/stylesheets/views/_homepage.scss]: I assume the `search-button.png` is used elsewhere?

**Comment by fofr** [comment]
Can you include screenshots for older browsers? Mainly IE, as I note the old code includes some strange IE fixes.

**Comment by andysellick** [comment]
Older browser screenshots added.

File: app/assets/stylesheets/views/_homepage.scss (hunk 1 above)
Comment (andysellick) [inline app/assets/stylesheets/views/_homepage.scss]: I do not believe it is, so it has been removed.

**Comment by fofr** [comment]
Static has been deployed so this can be merged when ready.

**Pull request merged**